mllib/src/test/scala/org/apache/spark/ml/feature/ChiSqSelectorSuite.scala

```diff
@@ -82,6 +82,18 @@ class ChiSqSelectorSuite extends SparkFunSuite with MLlibTestSparkContext
         case Row(vec1: Vector, vec2: Vector) =>
           assert(vec1 ~== vec2 absTol 1e-1)
       }
+
+    selector.setSelectorType("fwe").setAlpha(0.5).fit(df2).transform(df2)
+      .select("filtered", "preFilteredData").collect().foreach {
+      case Row(vec1: Vector, vec2: Vector) =>
+        assert(vec1 ~== vec2 absTol 1e-1)
+    }
+
+    selector.setSelectorType("fdr").setAlpha(0.21).fit(df2).transform(df2)
+      .select("filtered", "preFilteredData").collect().foreach {
+      case Row(vec1: Vector, vec2: Vector) =>
+        assert(vec1 ~== vec2 absTol 1e-1)
+    }
```

Comment:
Enforce this test case to large dimension data, and output different selected features according to selector type as far as possible.